<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectCreationForm</class>
 <widget class="QWidget" name="ProjectCreationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>525</width>
    <height>239</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProjectCreationForm</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Raw directory</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>18</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="rawDirPathText"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectRawDirBtn">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cooked directory</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>3</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="cookedDirPathText"/>
    </item>
    <item>
     <widget class="QPushButton" name="selectCookedDirBtn">
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project creation form</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Project file</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Preferred</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>35</width>
        <height>14</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLineEdit" name="projectFilePathText"/>
    </item>
    <item>
     <widget class="QPushButton" name="createProjectFileBtn">
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>501</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelBtn">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="icon">
       <iconset theme="cancel"/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>lineEdit</zorder>
  <zorder>lineEdit_3</zorder>
  <zorder>createProjectFileBtn</zorder>
  <zorder>projectFilePathText</zorder>
  <zorder>label</zorder>
  <zorder>createProjectFileBtn</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>label_4</zorder>
  <zorder>saveBtn</zorder>
  <zorder>cancelBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>projectFilePathText</tabstop>
  <tabstop>createProjectFileBtn</tabstop>
  <tabstop>rawDirPathText</tabstop>
  <tabstop>selectRawDirBtn</tabstop>
  <tabstop>cookedDirPathText</tabstop>
  <tabstop>selectCookedDirBtn</tabstop>
  <tabstop>cancelBtn</tabstop>
  <tabstop>saveBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>ProjectCreationForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
